airflow/providers/salesforce/operators/tableau_refresh_workbook.py

```diff
@@ -14,7 +14,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-from typing import Optional
+from typing import Optional, Any, Dict
```

Comment:
Do we need this imports, looks like Any and Dict are not used in this file

Reply:
forgot to optimize the import :D